Plugins/NativeEngine/Source/VertexBuffer.h

```diff
@@ -42,69 +42,20 @@ namespace Babylon
 
         static void BuildInstanceDataBuffer(bgfx::InstanceDataBuffer& instanceDataBuffer, const std::map<bgfx::Attrib::Enum, InstanceInfo>& instances, uint32_t instanceCount);
 
-    private:
-        void Build(uint32_t numVertices);
-
-        class Handle final
```

Comment:
Is this just removed because there aren't enough places doing this to make it worth encapsulating into its own class?

Reply:
Right. I could add it back, but it seems overkill.